src/app/user/add-user.component.ts

```diff
@@ -0,0 +1,36 @@
+import { FormGroup, FormControl } from '@angular/forms';
+import { Component, OnInit } from '@angular/core';
+@Component({
+  selector: 'app-add-user',
+  templateUrl: './add-user.component.html',
+  styleUrls: ['./add-user.component.css']
+})
+export class AddUserComponent implements OnInit {
+  userForm: FormGroup;
+  public UserName: string='';
+  constructor() { }
+  public userArray: string[] = [];
+
+addData(e){
+
+  this.userArray.push(this.UserName);
+  localStorage.setItem('userArray',JSON.stringify(this.userArray));
+  localStorage.getItem('local');
+  this.UserName= ''
+  
+}
+getData(){
+  if(localStorage.getItem('userArray')== null){
```

Comment:
array is already set to empty, no need of if condition,
can be written if(!null) condition